src/hooks/use-load-movies/use-load-movies.test.js

```diff
@@ -0,0 +1,33 @@
+import { renderHook } from '@testing-library/react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useLoadMovies } from './use-load-movies';
+import { useGetParams } from '../use-get-params/use-get-params';
+
+jest.mock('react-redux', () => ({
+  ...jest.requireActual('react-redux'),
+  useSelector: jest.fn(),
+  useDispatch: jest.fn(),
+}));
+
+jest.mock('../use-get-params/use-get-params', () => ({
+  useGetParams: jest.fn(),
+}));
+
+const mockState = {
+  movies: [],
+  isLoading: false,
+  error: undefined,
+  totalAmount: 0,
+}
+
+describe('useLoadMovies', () => {
+  it('Returns movie data', () => {
```

Comment:
Well, it's not movie data, it's rather movieState.

Reply:
Changed it to `'Returns movie state'`.